main/mainTemplate.json

```diff
@@ -70,7 +117,7 @@
         "description": "Virtual network address prefix."
```

Comment:
We need to change the default vale of Virtual Network to 10.0.0.0/16, as maximum number of Private IP Address per Virtual Network can be 65536 and this should resolve the warning "Selected subscription doesn't have have enough available address space that meets all the constraints of the default virtual network specified." which we get as of now while creating deployment through offer.

Reply:
A customer may have a different subscription.